README.md

```diff
@@ -59,31 +59,23 @@ ringpop.on('changed', function() {
 });
 ```
 
-# Forwarding a request
-Ringpop will typically be used by handing over request routing to it. As described earlier, the "process or forward" pattern is used to decide whether a request should be processed by the node that received the request or by another node. As an alternative, `handleOrForward` can be used to encapsulate that repetitive pattern. Here's an example of its use:
+# Request Proxying
+ringpop provides request routing as a convenience. When a request arrives at your services' public endpoints, you'll want to use ringpop to decide whether request processing should take place on the node that received the request or elsewhere. If elsewhere, ringpop will proxy your request to the correct destination.
 
-```javascript
-// Let's say this is an endpoint handler in your web application
-function endpoint(incoming, opts, cb) {
-    function handle() {
-        // process request
-    }
-
-    function forwarded(err, resp, body) {
-        cb(err, {
-            statusCode: resp && resp.statusCode
-        });
-    }
-
-    var requestToForward = {
-        method: 'POST',
-        path: '/supply/1',
-        headers: { 'Content-Type: application/json' },
-        body: JSON.stringify({ /* fill body here */ }),
-        timeout: 1000
-    };
-
-    ringpop.handleOrForward(incoming.params.uuid, handle, requestToForward, forwarded);
+Upon arrival of a proxied request at its destination, membership checksums of the sender and receiver will be compared. The request will be refused if checksums do not match. Mismatches are to be expected when nodes are entering or exiting the cluster due to deploys, added/removed capacity or failures. The cluster will eventually converge on one membership checksum, therefore, refused requests are best handled by retrying them.
+
+ringpop's request proxy has retries built in and can be tuned using 2 parameters provided at the time ringpop is instantiated: `requestProxyMaxRetries` and `requestProxyRetrySchedule` or per-request with: `maxRetries` and `retrySchedule`. The first parameter is an integer representing the number of times a particular request is retried. The second parameter is an array of integer or floating point values representing the delay in between consecutive retries.
+
+ringpop has codified the aforementioned routing pattern in the `handleOrProxy` function. It returns true when `key` hashes to the "current" node and false otherwise. `false` results in the request being proxied to the correct destination. Its usage looks like this:
+
+```js
+var opts = {
```

Comment:
yes absolutely. i though the same after creating the pr. will add it in tomorrow.

Sent from my iPhone

> On Feb 23, 2015, at 7:28 PM, Raynos (Jake Verbaten) <notifications@github.com> wrote:
> 
> In README.md:
> 
> > -        method: 'POST',
> > -        path: '/supply/1',
> > -        headers: { 'Content-Type: application/json' },
> > -        body: JSON.stringify({ /* fill body here */ }),
> > -        timeout: 1000
> > -    };
> > -
> > -    ringpop.handleOrForward(incoming.params.uuid, handle, requestToForward, forwarded);
> > +Upon arrival of a proxied request at its destination, membership checksums of the sender and receiver will be compared. The request will be refused if checksums do not match. Mismatches are to be expected when nodes are entering or exiting the cluster due to deploys, added/removed capacity or failures. The cluster will eventually converge on one membership checksum, therefore, refused requests are best handled by retrying them.
> > +
> > +ringpop's request proxy has retries built in and can be tuned using 2 parameters provided at the time ringpop is instantiated: `requestProxyMaxRetries` and `requestProxyRetrySchedule` or per-request with: `maxRetries` and `retrySchedule`. The first parameter is an integer representing the number of times a particular request is retried. The second parameter is an array of integer or floating point values representing the delay in between consecutive retries.
> > +
> > +ringpop has codified the aforementioned routing pattern in the `handleOrProxy` function. It returns true when `key` hashes to the "current" node and false otherwise. `false` results in the request being proxied to the correct destination. Its usage looks like this:
> > +
> > +```js
> > +var opts = {
> can we add statsd for retries. This allows us to see on a dashboard how much retrying ringpop does internally for proxying.
> 
> In fact adding statsd full stop so we can see proxy health in statsd would be super useful.
> 
> —
> Reply to this email directly or view it on GitHub.
>